### ISSUE TITLE
End-game leaderboard stretches on ultrawide

### ISSUE THREAD

**Issue by TheLetterZed** [Vexcenot/-Middle-Mann#120]
** Fill these checkmarks before reporting a bug!**
- [x] i am not using any additional mods that might overwrite any hud files (things like portraits and stuff of the sort)

**Describe the Issue**
When the match finishes, the end-game leaderboard stretches to fill the entire screen on ultrawide.

**To Reproduce (gamemode/loadout etc)**
1. Play a casual game with an ultra-wide resolution (eg. 3440x1440)
2. Wait for the match to finish
3. The end-game leaderboard stretches to fill the whole screen.

**Expected behavior**
I expect this to have a similar size to how it looks on a regular resolution. It should not expand to become wider on ultrawide screens. 

**Screenshots**
![image](https://github.com/Vexcenot/-Middle-Mann/assets/153703497/2948bd08-1482-4f63-b116-133653298301)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Resolution (and Aspect Ratio, eg. 1280x960 4:3): 3440x1080, 

**TF2 Specific Info (please complete the following information):**
 - HUD Version (either commit # or release version): 
3d27916
 - Customizations Applied (if any, if none type "none"): none
 - Other Mods in tf/custom? if so, which ones?: none

**Comment by TechnoSL** [comment]
i would try and recreate this but sadly i do not own a ultrawide, have you tried playing a full match on default hud to see if the end-game lb stretches on it too?

**Comment by TheLetterZed** [comment]
Yeah, it's the same on vanilla. I guess this is more of a feature request then. 😅 

![image](https://github.com/Vexcenot/-Middle-Mann/assets/153703497/09b85871-f20a-4bea-b8cb-1c1f0e1bcf15)

**Comment by TheLetterZed** [comment]
I think we can add this with the MatchSummary field in hudlayout.res:

https://github.com/Vexcenot/-Middle-Mann/blob/b6d22046ef9d0d4fc6de38b215ddc0a6aaab7f99/scripts/hudlayout.res#L1307-L1316

I'm not sure what values to pick for width and height - ideally this is restricted to a square or a 16:9 rectangle centered in the middle. Wasn't sure which [sizing variable](https://github.com/JarateKing/TF2-Hud-Reference/blob/master/1-APPENDIX/Sizing.md) made sense here. I'll play around with it next time I'm playing.

**Comment by TheLetterZed** [comment]
Could also incorporate this kind of fix so that the scoreboard isn't cut off at the top too: https://steamcommunity.com/sharedfiles/filedetails/?id=3116281066

**Comment by TechnoSL** [comment]
probably doesent fix the ultrawide stuff but might fix the scoreboard cutoff

cant test rn because i dont feel like playing a full casual match

**Comment by tekunotri** [comment]
closing due to nothing being done, maybe some other day but not currently
